platform/platformsh.css

```diff
@@ -263,14 +264,23 @@ aside {
 
 .reveal code {
   font-family: var(--brand-font-mono), monospace;
+  background: rgb(48, 47, 69);
+  color: #fff;
+  border-radius: 2px;
+}
+.hljs-ln-line .hljs-comment {
+  color: #fff;
+}
+.hljs-ln-line > .hljs-comment {
+  opacity: 0.6;
 }
 
 /* This is for inline code */
 .reveal :not(pre) > code {
-  color: var(--color-code);
-  background-color: var(--color-code-background);
-  padding: 0.1em;
-  border-radius: 3px;
+  color: var(--color-link);
```

Comment:
Making links and code the same color seems like a bad idea.  That's misleading as to which it is.  Links should always be a unique visual treatment.

Reply:
I had this styled to match the UI inline code, but updated it to match the website's